CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ include(CMakeDependentOption)
 
 if (NOT MSVC)
     include(FindPkgConfig)
+    set(CMAKE_PREFIX_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake;${CMAKE_MODULE_PATH}")
```

Comment:
Why is MSVC excluded here…

Reply:
Because it's the CMake file from MSVC, as the commit description states.

Reply:
Leave a comment in there so it's clear